docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/quote/quote.scss

```diff
@@ -6,7 +6,12 @@
   $root: &;
 
   @include ct-reset-box-model();
-  @include ct-highlight-stripe($ct-quote-border-width, 0, 0);
+  @include ct-highlight-stripe($ct-quote-border-width, $ct-quote-border-radius, 0);
+
+  &::before {
```

Comment:
@govindmaloo 
this is a defect in design - the strip should have no rounded corners. can you please remove `$ct-quote-border-radius` variables and use `0`